packages/migrate/src/types.ts

```diff
@@ -369,12 +370,17 @@ export namespace EngineArgs {
     affected: AffectedViewAndField[]
   }
 
-  // Partioned Tables
+  // Partitioned Tables
```

Comment:
Do we even need this comment? It doesn't seem to add much information here.

Reply:
ope, it gets updated in another PR. when I had originally added partitioned tables, I figured there were going to be more warnings, but it's actually just going to be the one per stopgap. [Comparison](https://github.com/prisma/prisma/pull/18654/files#diff-2aeb8416b944a3e8efa6ac5e85341d7bfe5ac46b3557e7a083d43c2b5ab46321)